packages/tx/test/types.ts

```diff
@@ -66,3 +66,61 @@ export type OfficialTransactionTestData = {
   result: ForksData
   txbytes: string
 }
+
+export interface EnvData {
+  currentBaseFee: string
+  currentCoinbase: string
+  currentDifficulty: string
+  currentExcessBlobGas: string
+  currentGasLimit: string
+  currentNumber: string
+  currentRandom: string
+  currentTimestamp: string
+}
+
+export interface AccountState {
+  balance: string
+  code: string
+  nonce: string
+  storage: Record<string, string>
+}
+
+export interface Indexes {
+  data: number
+  gas: number
+  value: number
+}
+
+export interface PostReceipt {
+  hash: string
+  indexes: Indexes
+  logs: string
+  txbytes: string
+}
+
+export interface BlobTransaction extends TxData {
+  accessLists: Array<{ address: string; storageKeys: string[] }>
+  blobVersionedHashes: string[]
+  maxFeePerBlobGas: string
+  maxFeePerGas: string
+  maxPriorityFeePerGas: string
+  secretKey: string
+  sender: string
+  to: string
+}
+
+export type OfficialStateTestData = {
+  _info: {
+    comment: string
+    generatedTestHash: string
+    filledwith: string // cspell:disable-line
+    lllcversion: string // cspell:disable-line
+    solidity: string
+    source: string
+    sourceHash: string
+  }
+  env: EnvData
+  pre: Record<string, AccountState>
+  post: Partial<Record<ForkName, PostReceipt[]>>
+  transaction: BlobTransaction
+}
```

Comment:
These types are added with the following commit message "Add types for vm state test data", so with "vm" in the text, while the types here are in "tx", so there seems to be something off.

Reply:
Yes, good point, they would have had to have been moved over to the `vm` package along with making `getTests` and other helpers in `testLoader.ts` available there too for use with the new runners. But we could leave that as a potential followup.

Reply:
Side note if we/you tackle later: these tx specific things should *never* make it over, there should definitely no unified **runner**, only a very very lightweight unified 20 line file **loader** (as outlined in the chat) helper function, which is completely independent of whatever VM or TX code and can therefore be placed in a generic place (`testdata` package).